docs/3.0.0-alpha.x/getting-started/quick-start-tutorial.md

```diff
@@ -10,10 +10,9 @@ By following this tutorial, we will get Strapi installed globally onto your syst
 
 ::: tip NOTE
 
-You need to have ***Node.js and npm*** installed on your system prior to following these steps. If you do not have Node.js and npm installed or are not sure, please visit our [Installation Requirements](http://localhost:8080/documentation/3.0.0-alpha.x/getting-started/install-requirements.html).
+You need to have **_Node.js and npm_** installed on your system prior to following these steps. If you do not have Node.js and npm installed or are not sure, please visit our [Installation Requirements](http://localhost:8080/documentation/3.0.0-alpha.x/getting-started/install-requirements.html).
```

Comment:
Shouldn't we use this URL https://strapi.io/documentation/3.0.0-alpha.x/getting-started/install-requirements.html instead?

Reply:
oops yes yes but no.  [Installation Requirements](/3.0.0-alpha.x/getting-started/install-requirements.html) Is the best way, but nonetheless this is fixed.